http/org.wso2.carbon.transport.http.netty/src/test/java/org/wso2/carbon/transport/http/netty/util/server/HttpsServer.java

```diff
@@ -28,99 +28,78 @@
 import org.slf4j.LoggerFactory;
 import org.wso2.carbon.transport.http.netty.util.TestUtil;
 
+import java.io.FileInputStream;
 import java.net.InetSocketAddress;
+import java.security.KeyStore;
+import javax.net.ssl.KeyManagerFactory;
 import javax.net.ssl.SSLContext;
 
 /**
  * A Simple HTTP Server
  */
-public class HTTPServer {
+public class HttpsServer implements TestServer {
 
-    private static final Logger logger = LoggerFactory.getLogger(HTTPServer.class);
+    private static final Logger logger = LoggerFactory.getLogger(HttpsServer.class);
 
-    private int port = 9000;
+    private int port = 9002;
```

Comment:
What is the reason for using 9002 here? In the utility class, it is used 9004 as HTTPS port and 9000 as HTTP port.